internal/events/aggregator.go

```diff
@@ -428,10 +428,10 @@ func (ag *aggregator) attemptMessageDispatch(ctx context.Context, msg *fftypes.M
 	valid := true
 	eventType := fftypes.EventTypeMessageConfirmed
 	switch {
-	case msg.Header.Namespace == fftypes.SystemNamespace:
+	case msg.Header.Type == fftypes.MessageTypeDefinition:
```

Comment:
This looks right to me 👍
Think this is the substantial change, matching the package rename.
We went from "If it's on the system namespace, the system handlers handle it"...
To "If it's a definition, then the definition handlers handle it"